oas/3.1/schema-base/2021-05-20

```diff
@@ -0,0 +1,24 @@
+{
+  "$id": "https://spec.openapis.org/oas/3.1/schema-base/2021-05-20",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$ref": "https://spec.openapis.org/oas/3.1/schema/2021-05-20",
+  "properties": {
+    "jsonSchemaDialect": {
+      "$ref": "#/$defs/dialect"
+    }
+  },
+  "$defs": {
+    "dialect": {
+      "const": "https://spec.openapis.org/oas/3.1/dialect/base"
+    },
+    "schema": {
+      "$dynamicAnchor": "meta",
+      "$ref\"": "https://spec.openapis.org/oas/3.1/dialect/base",
```

Comment:
This is an error that I thought had already been fixed?

Reply:
Good catch. I thought that was fixed too.

Reply:
Ahh, it's because #2545 hasn't been merged yet and I copied the bug from original.